sql/create_shards.sql.tmpl

```diff
@@ -79,3 +79,32 @@ SELECT sort_names('sumedh', 'jason', 'ozgun');
 SELECT create_table_then_fail('localhost', $PGPORT);
 
 SELECT COUNT(*) FROM pg_class WHERE relname LIKE 'throwaway%' AND relkind = 'r';
+
+\set VERBOSITY terse
+
+-- test foreign table creation
+CREATE FOREIGN TABLE foreign_table_to_distribute
+(
+	name text,
+	id bigint
+)
+SERVER fake_fdw_server;
+
+SELECT master_create_distributed_table('foreign_table_to_distribute', 'id');
+SELECT master_create_worker_shards('foreign_table_to_distribute', 16, 1);
+
+\set VERBOSITY default
+SELECT storage, min_value, max_value FROM pgs_distribution_metadata.shard
+	WHERE relation_id = 'foreign_table_to_distribute'::regclass
+	ORDER BY (min_value COLLATE "C") ASC;
+
+-- cleanup foreign table, related shards and shard placements
```

Comment:
I had been adding e.g. `DROP TABLE` statements to tests as necessary to ensure each file ran on a mostly clean database, but hadn't thought about the things in `pgs_distribution_metadata`. I'm not sure whether we want to clean it out between each test or not, but either way we'll want some unification across our tests. You don't need to change anything here, I just wanted to make a note to myself that we need a little bit of cleanup in this regard.